invenio_saml/handlers.py

```diff
@@ -66,7 +66,7 @@ def account_info(attributes, remote_app):
         external_id=external_id,
         external_method=remote_app,
         active=True,
-        confirmed_at=datetime.utcnow().isoformat()
+        confirmed_at=datetime.utcnow()
```

Comment:
What is the reason of changing the confirmed_at date format?

Reply:
Good question; I forgot to write a comment myself.
When running the tests, I encounter an issue with this and SQLite, something to do with being unable to parse dates from strings.
But now that you point this out, SQLite is not part of the test matrix, so we can probably put the formatted output back.

Reply:
https://github.com/inveniosoftware/invenio-accounts/pull/428#discussion_r1024914387

ok we go with `datetime.utcnow()`